osu.Game/Screens/Play/EpilepsyWarning.cs

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Audio;
+using osu.Framework.Audio.Track;
+using osu.Framework.Bindables;
+using osu.Framework.Extensions.Color4Extensions;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using osu.Framework.Graphics.Sprites;
+using osu.Game.Beatmaps;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Containers;
+using osuTK;
+using osuTK.Graphics;
+
+namespace osu.Game.Screens.Play
+{
+    public class EpilepsyWarning : VisibilityContainer
+    {
+        private readonly BindableDouble trackVolumeOnEpilepsyWarning = new BindableDouble(1f);
+
+        private Track track;
+        private FillFlowContainer warningContent;
+
+        public EpilepsyWarning()
+        {
+            RelativeSizeAxes = Axes.Both;
+            Alpha = 0f;
+        }
+
+        [BackgroundDependencyLoader]
+        private void load(OsuColour colours, IBindable<WorkingBeatmap> beatmap)
+        {
+            Children = new Drawable[]
+            {
+                new Box
```

Comment:
not sure we want to do this. adding +1 fill rate while displayed.

Reply:
hmm, the screen background could expose a bindable to dim it or otherwise. but could also remove the dim effect and call it a day.